basic-nginx/Jenkinsfile

```diff
@@ -55,50 +55,17 @@ pipeline {
       }
     }
 
-    stage('Verify Deployment to Dev') {
```

Comment:
@kenwilli ; for the verify bits. can you try moving to version 1.6 of the pipeline lib. A fix (Update verify to look at dc rollout status) was added that sounds like it should stop the verify issue you are mentioning.
- https://github.com/redhat-cop/pipeline-library/releases/tag/1.6